IOPool/Input/src/RepeatingCachedRootSource.cc

```diff
@@ -369,6 +415,30 @@ void RepeatingCachedRootSource::readProcessBlock_(ProcessBlockPrincipal& process
   rootFile_->readProcessBlock_(processBlockPrincipal);
 }
 
+WrapperBase const* RepeatingCachedRootSource::RCProductGetter::getIt(ProductID const& iPID) const {
+  auto itFound = map_->find(iPID);
+  if (itFound == map_->end()) {
+    return nullptr;
+  }
+  return (*wrappers_)[itFound->second].get();
+}
+
+std::optional<std::tuple<WrapperBase const*, unsigned int>>
+RepeatingCachedRootSource::RCProductGetter::getThinnedProduct(ProductID const&, unsigned int key) const {
+  return {};
+};
```

Comment:
Do these imply that `RepeatingCachedRootSource` does not support thinned products?

Reply:
Correct. As that feature isn't used in production and is most likely not being used in the 'wild' I didn't feel it was worth the effort to try to get to work.

Reply:
I tend to agree. How about adding a comment somewhere here that this feature is intentionally omitted for now, and can be added later if there is use for it? (to avoid the need to wonder in the future if the case was that or if there was some fundamental reason)

Reply:
Done